### PR TITLE
replay: capacitor

### PR DESCRIPTION
**Pull request by bruno-garcia**
Docs for Session replay for capacitor

### CODE REVIEW

File: src/platform-includes/session-replay/setup/javascript.capacitor.mdx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+To set up the integration, add the following to your Sentry initialization. Several options are supported and passable using the integration constructor. See the [configuration documentation](/javascript/guides/capacitor/session-replay/configuration/) for more details.
+
```
Comment (bruno-garcia) [inline src/platform-includes/session-replay/setup/javascript.capacitor.mdx:2]: The minimal supported version of the Sentry SDK for Capacitor is 0.11.0.

Comment (bruno-garcia) [inline src/platform-includes/session-replay/setup/javascript.capacitor.mdx:2]:
```suggestion

The minimal supported version of the Sentry SDK for Capacitor is 0.11.0.

```

File: src/platform-includes/session-replay/setup/javascript.capacitor.mdx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+To set up the integration, add the following to your Sentry initialization. Several options are supported and passable using the integration constructor. See the [configuration documentation](/javascript/guides/capacitor/session-replay/configuration/) for more details.
```
Comment (lizokm) [inline src/platform-includes/session-replay/setup/javascript.capacitor.mdx:1]:
```suggestion
Several options are supported and passable using the integration constructor. See the [configuration documentation](/javascript/guides/capacitor/session-replay/configuration/) for more details. To set up the integration, add the following to your Sentry initialization:
```

File: src/platform-includes/session-replay/setup/javascript.capacitor.mdx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+The minimal supported version of the Sentry SDK for Capacitor is 0.11.0.
```
Comment (lizokm) [inline src/platform-includes/session-replay/setup/javascript.capacitor.mdx:3]:
```suggestion
<Note>
The minimal supported version of Sentry's SDK for Capacitor is 0.11.0.
</Note>
```

File: src/platform-includes/session-replay/setup/javascript.capacitor.mdx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+```javascript {tabTitle: ESM}
+import * as Sentry from '@sentry/capacitor';
+import { Replay } from "@sentry/replay";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+
+  // This sets the sample rate to be 10%. You may want this to be 100% while
+  // in development and sample at a lower rate in production
+  replaysSessionSampleRate: 0.1,
```
Comment (lizokm) [inline src/platform-includes/session-replay/setup/javascript.capacitor.mdx:14]:
```suggestion
  // This sets the sample rate at 10%. You may want this to be 100% while
  // in development, then sample at a lower rate in production.
  replaysSessionSampleRate: 0.1,
```

File: src/platform-includes/session-replay/setup/javascript.capacitor.mdx [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+      // Additional SDK configuration goes in here, for example:
+      maskAllText: true,
+      blockAllMedia: true,
+    }),
+  ],
+});
+```
+
+With the settings above, session replays with errors are always captured.
+You can verify that with the following:
```
Comment (lizokm) [inline src/platform-includes/session-replay/setup/javascript.capacitor.mdx:31]:
```suggestion
Session replays with errors, will always be captured with the settings above.
Verify it with the following:
```

File: src/wizard/javascript/replay-onboarding/capacitor/2.configure.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+name: JavaScript
+doc_link: https://docs.sentry.io/platforms/javascript/guides/capacitor/session-replay/
+support_level: production
+type: language
+---
+
+#### Configure
+
+Add the following to your SDK config. There are several privacy and sampling options available, all of which can be set using the `integrations` constructor. Learn more about configuring Session Replay by reading the [configuration docs](https://docs.sentry.io/platforms/javascript/session-replay/).
+
```
Comment (lizokm) [inline src/wizard/javascript/replay-onboarding/capacitor/2.configure.md:11]:
```suggestion
There are several privacy and sampling options available, all of which can be set using the `integrations` constructor. Learn more about configuring Session Replay by reading the [configuration docs](https://docs.sentry.io/platforms/javascript/session-replay/). Add the following to your SDK config:

```

File: src/wizard/javascript/replay-onboarding/capacitor/2.configure.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+```javascript
+import * as Sentry from "@sentry/capacitor";
+import { Replay } from "@sentry/replay";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+
+  // This sets the sample rate to be 10%. You may want this to be 100% while
+  // in development and sample at a lower rate in production
```
Comment (lizokm) [inline src/wizard/javascript/replay-onboarding/capacitor/2.configure.md:20]:
```suggestion
  // This sets the sample rate at 10%. You may want this to be 100% while
  // in development, then sample at a lower rate in production.
```

**Review by lizokm — APPROVED** [review]
A few minor tweaks, but otherwise LGTM

File: src/platform-includes/session-replay/setup/javascript.capacitor.mdx [hunk 8]
```diff
@@ -0,0 +1,37 @@
+Several options are supported and passable using the integration constructor. See the [configuration documentation](/javascript/guides/capacitor/session-replay/configuration/) for more details. To set up the integration, add the following to your Sentry initialization:
```
Comment (bruno-garcia) [inline src/platform-includes/session-replay/setup/javascript.capacitor.mdx:1]:
```suggestion
Several options are supported and passable using the integration constructor. See the [configuration documentation](/platforms/javascript/guides/capacitor/session-replay/configuration/) for more details. To set up the integration, add the following to your Sentry initialization:
```

**Comment by lucas-zimerman** [comment]
Other than the link issue the doc seems ok. LGTM!

**Pull request merged**